### DISCUSSION TITLE
Minimum Specs

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [immense/Remotely#708] [General]
Installed Remotely using Docker Compose and connected fine to a remote machine. However things got slow very quickly with eventually the screen not refreshing. 

What are considered the minimum specs for the server.

Would working through a Cloudflare tunnel make things unworkable?

**Comment by gabrielporto-mkt** [comment]
> Installed Remotely using Docker Compose and connected fine to a remote machine. However things got slow very quickly with eventually the screen not refreshing.
> 
> What are considered the minimum specs for the server.
> 
> Would working through a Cloudflare tunnel make things unworkable?

I use cloudflare tunnel and it works just fine. My container on proxmox has 1 core and 1.5GB of RAM.
